specs/packagemanager/PackageManagement.md

```diff
@@ -125,6 +127,7 @@ The following table shows the supported permutations of verbs and targets:
 |Remove                  |   X    |    X     |        WAS        |      OS/WAS     |   X    |    X     | OS/WAS  |    WAS     |
 |Repair                  |   X    |    X     |        WAS        |       WAS       |   X    |    X     |  WAS    |    WAS     |
 |Reset                   |   X    |    X     |        WAS        |       WAS       |   X    |    X     |  WAS    |    WAS     |
+|IsProvisioned           |   X    |    X     |       OS/WAS      |      OS/WAS     |   X    |    X     |  WAS    |    WAS     |
```

Comment:
Provisioning is based on package family.

RECOMMEND: Change table marking IsProvisioned/PackageFullName=X (not supported)